jina/helper.py

```diff
@@ -721,6 +721,7 @@ def kwargs2namespace(
         parser: ArgumentParser,
         warn_unknown: bool = False,
         fallback_parsers: List[ArgumentParser] = None,
+        positional_args: Tuple[str, ...] = None,
```

Comment:
what is an example where this is used?

Reply:
```suggestion
        positional_args: Optional[Tuple[str, ...]] = None,
```

Reply:
I would keep `positional_args` outside, the user can append them on its own. The name of the function explicitly says `kwargs2Namespace` so why would it have to take `positional_args`?

Reply:
because parser won't work

Reply:
i would keep it as is as other wise people have to do this aganin

Reply:
> what is an example where this is used?
in this pR